src/extensions/nativeExtensions/image/Image.ts

```diff
@@ -27,24 +31,7 @@ export default class Image extends AbstractExtension {
             [VuetifyIconsGroups.mdi]: new VuetifyIcon('mdi-image'),
             [VuetifyIconsGroups.mdiSvg]: new VuetifyIcon('M8.5,13.5L11,16.5L14.5,12L19,18H5M21,19V5C21,3.89 20.1,3 19,3H5A2,2 0 0,0 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19Z')
           },
-          nativeExtensionName,
-          async onClick ({ context, editor }) {
-            const LinkWindowComponent = Vue.extend(ImageWindow)
-            const instance = new LinkWindowComponent({
-              vuetify: Vue.prototype.tiptapVuetifyPlugin.vuetify,
```

Comment:
@avgeeklucky You won’t believe it, but it turns out we just needed to write `parent: this` here and the problem would be solved.

I found out about it from [here](
https://css-tricks.com/creating-vue-js-component-instances-programmatically/#comment-1614914).

I checked and it works.

In that case, I would like to leave the previous way, as it was before.

The previous version is more familiar to me and seems more flexible. Sorry for the fact that you did nothing in vain, but I found a solution in one line.